app/controllers/admin/stats_controller.rb

```diff
@@ -0,0 +1,70 @@
+# -*- encoding : utf-8 -*-
+
+class Admin::StatsController < ApplicationController
+  def index
+    # total user count
+    @users_count = User.count
+
+    # The most recent users to join
+    @users_stats = params.fetch(:users_stats, {})
+    @recent_users = recent_users
+
+    # Query Solr for top depositors
+    @active_users = top_depositors
+
+    # Count of documents by permissions
+    @files_count = document_by_permission
+
+    # Query Solr for top file formats
+    @top_formats = top_formats
+
+    render 'index'
+  end
+
+  private 
+  def top_depositors_count
+    count = params[:dep_count].to_i 
+    count.in?(5..20) ? count : 5
+  end
+
+  def document_by_permission
+    # make sure everything is up to date in solr
+    ActiveFedora::SolrService.instance.conn.commit("expungeDeletes"=>true)
```

Comment:
From what I can tell in the solr documentation, expungeDeletes has to do with how the index is structured on disk, but shouldn't change the returned results, so can we remove this?